config/crd/bases/api.storageos.com_pools.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.0
+  creationTimestamp: null
+  name: pools.api.storageos.com
+spec:
+  group: api.storageos.com
+  names:
+    kind: Pool
+    listKind: PoolList
+    plural: pools
+    singular: pool
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        description: Pool is the Schema for the pools API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PoolSpec defines the desired state of Pool
+            properties:
+              nodeDriveMap:
+                additionalProperties:
+                  items:
+                    type: string
+                  type: array
+                description: NodeDriveMap maps which drives on which nodes are in
+                  the pool. A node is only in the pool if it has at least one drive
+                  that is in the pool
+                type: object
+            type: object
+          status:
+            description: PoolStatus defines the observed state of Pool
```

Comment:
I guess this confirms "status" is meant to be "state"?
Unless k8s makes this a pattern and should be named that way...
Let me know the reasoning behind it 🙏

Reply:
Status is a k8s thing. Each object has Metadata (name, namespace, labels, etc), a Spec (what the CR is meant to be) and a Status (what the CR currently is). 

In our case we're going to use the status to give a more user-friendly way of seeing which drives are in the pool.